wp_nav:
  pid_x:
    kp: 0.3
    ki: 0
    kd: 0
    i_a: 0
    o_t: 0.3
    tune_range: [1.0, 0.1, 1.0, 0.8, 1.0]  # pid调节器范围

  pid_y:
    kp: 0.3
    ki: 0
    kd: 0
    i_a: 0
    o_t: 0.3
    tune_range: [1.0, 0.1, 2.0, 0.8, 1.0]  # pid调节器范围

  pid_yaw:
    kp: 0.4
    ki: 0
    kd: 0
    i_a: 0
    o_t: 30
    tune_range: [2.0, 0.1, 2.0, 40, 60]  # pid调节器范围

  exit_l: 0.02  # m
  exit_a: 2  # °
  exit_cnt: 5000